schema.sql

```diff
@@ -1,11 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
  CREATE TABLE animals(
-    id INT GENERATED ALWAYS AS IDENTITY, 
+    id INT, 
     name TEXT,
     date_of_birth DATE, 
     escape_attempts INT, 
     neutered BOOLEAN, 
     weight_kg DECIMAL,
+    species TEXT,
     PRIMARY KEY(id)
 );
```

Comment:
- [ ] Kindly make sure to add the query to alter the table. You have made changes to the existing one. You should add the query below the table as  `ALTER TABLE animals ADD species varchar(255);`